src/Manager.h

```diff
@@ -9,6 +9,8 @@
 #include <iostream>
 #include <vector>
 
+using namespace std;
```

Comment:
If you added this probably you can remove the "std::" from "std::string"/ "std::vector"

Reply:
ok